assets/js/modules/search-console/components/dashboard/DashboardImpressionsWidget.js

```diff
@@ -88,14 +89,35 @@ function DashboardImpressionsWidget() {
 		return getNoDataComponent( _x( 'Search Console', 'Service name', 'google-site-kit' ) );
 	}
 
-	const { totalImpressions, totalImpressionsChange, dataMap } = extractSearchConsoleDashboardData( data );
+	// const { totalImpressions, totalImpressionsChange, dataMap } = extractSearchConsoleDashboardData( data );
```

Comment:
Please clean this up 🙂

Reply:
I did this and didn't push the commit. 🤦